test/linalg/uniformscaling.jl

```diff
@@ -51,7 +51,7 @@ end
 end
 
 @testset "det and logdet" begin
-    @test det(I) === 1
+    @test det(I) === true
```

Comment:
This is the only bit that gives me pause; I wonder if we should have `prod([true,true,true]) === 1` instead of `true`. We already have `sum([true,true,true]) === 3` so it would kind of make sense for the type of `prod` to match, even though `0` and `1` would be the only possible value.

Reply:
Perhaps warrants a dedicated issue? :)

Reply:
Done: https://github.com/JuliaLang/julia/issues/24425